fract4dgui/preferences.py

```diff
@@ -1,11 +1,10 @@
 # GUI for user settings
 
+from . import dialog, utils
+from gi.repository import Gtk, GObject
```

Comment:
I would revert this move. For style with the project imports coming last, plus `gi.repository` needs gi and `require_version` comes before Gtk.

Reply:
I don't think I did that on purpose. Maybe autopep8? not sure